tests/test_constants_center_frequencies.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pytest
+from numpy import testing as npt
+import pyfar
+
+import pyfar.constants
+
+
+def test_center_frequencies_iec():
```

Comment:
```suggestion
def test_nominal_iec_octave_default():
```

every paragraph could be a single test with a name describing what it does. the advantage is, that if the test will fail at some point, it would be much faster to find the reason, due to the test which fails.

This way is also fine, so feel free do find your style.